docs/site/translation.md

```diff
@@ -105,17 +118,31 @@ please file a ticket. [CLDR-18283](https://unicode-org.atlassian.net/browse/CLDR
 ### DateTime formats
 
 #### Gregorian Calendar (Year First) calendar
-💡🆕 This is a variant of the Gregorian calendar whose formats always use year-month-day ordering and a 24-hour time cycle. See [Year First Calendar] for more details.
+🆕 2025-05-09 — This is a variant of the Gregorian calendar whose formats always use year-month-day ordering and a 24-hour time cycle. See [Year First Calendar] for more details.
 *Note: the code is `iso8601`, but disregard that; it will be changed after submission.*
 
 <!-- [CLDR-18447](https://unicode-org.atlassian.net/browse/CLDR-18447) -->
 
 #### New flexible format and interval patterns 
-💡🆕 There are some new patterns for you to supply. Make sure that the format is consistent with related patterns. 
+🆕 2025-05-09 — There are some new patterns for you to supply. Make sure that the format is consistent with related patterns. 
 
 Note: Some locales have inconsistent patterns using eras: in some patterns using G (AD vs BC in Gregorian) but in related patterns using GGGGG (which is a narrow form: A vs B in Gregorian).
 The GGGGG is not typically needed except in special cases, such as the Japanese calendar.
 
+**🆕 2025-05-28** — Please review the Gregorian [flexible date/time](https://st.unicode.org/cldr-apps/v#/USER/Gregorian/header_Formats_Flexible_Date_Formats) and [interval date/time formats](https://st.unicode.org/cldr-apps/v#/USER/Gregorian/header_Formats_Intervals_Date_Formats). There are inconsistencies between formats in many locales. New examples and warnings have been added to help you see the issues:
+
+Intervals have "Related Flexible Dates" that show inconsistencies between the intervals and flexible. For example, in the following the era (n.C.) is on the opposite side.
+
+![Screenshot 2025-05-25 at 17 35 42](https://github.com/user-attachments/assets/35ac8d16-2286-41c0-a978-9acb9b2aa6e5)
```

Comment:
best not to use  ephemeral github assets - should add the file locally under `docs/site/images/whatever.png` and then refer to it as `/images/whatever.png`

Reply:
it is very convenient to paste images into a markdown file in the github web interface, and have it show up. Is there a way to reset that so that the files go into docs/site/images/ automatically?

Reply:
Was this resolved? The link still seems to point elsewhere?

Reply:
I would like to go ahead with this, and make another ticket for fixing images.